datumaro/plugins/voc_format/extractor.py

```diff
@@ -42,20 +61,21 @@ def __init__(self, path, task, **kwargs):
             label_idx, None
         )
         log.debug(
-            "Loaded labels: %s"
-            % ", ".join(
+            "Loaded labels: %s",
+            ", ".join(
                 "'%s' %s" % (l.name, ("(%s, %s, %s)" % c) if c else "")
                 for i, l, c in (
                     (i, l, label_color(i))
                     for i, l in enumerate(self._categories[AnnotationType.label].items)
                 )
-            )
+            ),
         )
         self._items = {item: None for item in self._load_subset_list(path)}
 
-    def _get_label_id(self, label):
+    def _get_label_id(self, label: str) -> Optional[int]:
```

Comment:
`Optional` seems wrong, because this function can't return `None`.

Reply:
Fixed